system/datatypes/peer_reviewed_article.php

```diff
@@ -416,14 +429,49 @@ public function editAction(HyphaRequest $request) {
 			return ['redirect', $this->constructFullPath($this->pagename)];
 		}
 
+		$formData = $request->getPostData() + $_FILES;
+
 		// create form
-		$form = $this->createEditForm($request->getPostData());
+		$form = $this->createEditForm($formData);
+
+		$featuredImageData = $form->dataFor(self::FIELD_NAME_FEATURED_IMAGE);
+		$orgImg = $this->xml->documentElement->getOrCreate(self::FIELD_NAME_FEATURED_IMAGE)->getText();
+		$hasNewImg = $featuredImageData && !$featuredImageData['error'];
```

Comment:
If `$featuredImageData['error']` is set, then the upload is completely ignored, without a message to the user. Is that ok? When can this happen? Doesn't this include "File too big"?